evap/evaluation/tools.py

```diff
@@ -275,7 +275,10 @@ def send_publish_notifications(courses):
             publish_notifications[course.responsible_contributor].add(course)
 
     for user, course_set in publish_notifications.items():
-        EmailTemplate.send_publish_notifications_to_user(user, list(course_set))
+        if not template:
+            template = EmailTemplate.objects.get(name=EmailTemplate.PUBLISHING_NOTICE)
```

Comment:
you're right, i missed the other two places.

please put this line outside the for loop, preferably at the beginning of the function.